cookbook_se/Makefile

```diff
@@ -15,22 +15,20 @@ serialize_sandbox: docker_build
 	rm ${CURDIR}/_pb_output/* || true
 	docker run -v ${CURDIR}/_pb_output:/tmp/output ${FULL_IMAGE_NAME}:${VERSION} make in_container_serialize_sandbox
 
-.PHONY: in_container_fast_serialize_sandbox
-in_container_fast_serialize_sandbox:
-	pyflyte --config /root/sandbox.config serialize fast workflows -f /tmp/output --source-dir /root/recipes
 
 .PHONY: fast_register_sandbox
-fast_register_sandbox: docker_build fast_serialize_sandbox
+fast_register_sandbox: fast_serialize_sandbox
```

Comment:
maybe an echo here indicating that the registration is against the local running instance of flyte,. If you want to register with another instance then follow docs - x